plugins/edsm.py

```diff
@@ -124,20 +124,20 @@ def plugin_start3(plugin_dir: str) -> str:
     this._IMG_ERROR = tk.PhotoImage(data=IMG_ERR_B64)  # BBC Mode 5 '?'
 
     # Migrate old settings
-    if not config.get('edsm_cmdrs'):
-        if isinstance(config.get('cmdrs'), list) and config.get('edsm_usernames') and config.get('edsm_apikeys'):
+    if not config.get_list('edsm_cmdrs'):
+        if isinstance(config.get_list('cmdrs'), list) and config.get_list('edsm_usernames') and config.get_list('edsm_apikeys'):
```

Comment:
Technically there's no way config.get_list() can return other than `list` now, right ?  Actually, given this is some very old settings migration code, perhaps that first get_list() should still be get() ?  I think before 2.34 it was a simple string ?

Reply:
It'll throw an error if its not a list from the config side itself, otherwise the only other thing it can return in that form is `None` if the field does not exist